cmd/webhook/main.go

```diff
@@ -74,15 +75,18 @@ var types = map[schema.GroupVersionKind]resourcesemantics.GenericCRD{
 	resolutionv1beta1.SchemeGroupVersion.WithKind("ResolutionRequest"): &resolutionv1beta1.ResolutionRequest{},
 }
 
+var webhookName string
```

Comment:
NIT: perhaps use the following syntax for multiple vars?

```
var (

)
```

I wonder if there is any special reason for setting this as a global var as opposed to passing it into the `new*Controller` functions?

Reply:
Thanks @afrittoli for a great suggestion!

Dropped using global variable and added a `webhookName` which is being passed into `new*Controller` functions. Updated `new*Controller` functions such that the controller name can be passed from `main`.